sdk/ml/azure-ai-ml/azure/ai/ml/_restclient/v2023_04_01_preview/models/_azure_machine_learning_workspaces_enums.py

```diff
@@ -627,8 +627,8 @@ class ForecastingModels(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     #: This method is suitable for forecasting when data is stationary/non stationary, and
     #: multivariate with any type of data pattern, i.e., level/trend /seasonality/cyclicity.
     ARIMAX = "Arimax"
-    #: TCNForecaster: Temporal Convolutional Networks Forecaster. //TODO: Ask forecasting team for
-    #: brief intro.
+    #: TCNForecaster: Temporal Convolutional Networks Forecaster hierarchically captures relationships
+    #: at low-, intermediate- and high- time scales to produce the forecast. 
```

Comment:
Would change this to something like: Deep neural network model capable of modeling correlations over long time periods.

Reply:
Fixed.